dashboard/app/models/school_info.rb

```diff
@@ -290,4 +290,22 @@ def public_school?
   def charter_school?
     school_type.eql? SCHOOL_TYPE_CHARTER
   end
+
+  # Decides whether the school info is complete enough to stop bugging the
+  # teacher for additional information every week.  Different from complete
+  # record validation.
+  def complete?
+    return true unless school_id.nil?
```

Comment:
nit: this would be more readable to me as `return true if school_id` but if you don't agree, feel free to ignore me :)

Reply:
Updated, Thanks